django_feature_flip/helpers.py

```diff
@@ -25,12 +25,7 @@ def __getattr__(self, name):
     def enabled(self, feature_name, actor=None):
         feature = get_feature_or_raise_error(feature_name)
 
-        if feature.totally_enabled:
-            return True
-        elif actor:
-            return self.feature_enabled_for_a_group_of_actor(feature, actor)
-        else:
-            return False
+        return feature.totally_enabled or self.feature_enabled_for_actor(feature, actor)
 
     def register(self, group_name, func):
         Group.objects.get_or_create(name=group_name)
```

Comment:
Is defined `name` as unique in the `Group` model?
It's important to avoid troubles in the database with `get_or_create`